Formula/shtools.rb

```diff
@@ -15,9 +15,12 @@ class Shtools < Formula
   end
 
   depends_on "fftw"
-  depends_on "gcc"
   depends_on "openblas"
 
+  on_macos do
+    depends_on "gcc"
+  end
+
```

Comment:
This is wrong.  @carlocab the audit for Linux-only GCC dependencies is mistakenly flagging GCC dependencies that are shared on both platforms.

Reply:
Homebrew/brew#13840